src/actions/visits/ActionCreators.jsx

```diff
@@ -1,26 +1,26 @@
 import ApiService, { handleNetworkError, dispatchAction } from "../../middleware/ApiService";
-import { VISITS_DATA } from "./Actions";
+import { VISITSDATA } from "./Actions";
 
 // Fetch all visits data
 export const GET_VISITS = () => async (dispatch) => {
   try {
-    const apiResponse = await ApiService.get(`/visits`);
+    const apiResponse = await ApiService.get(`/appointments`);
     if (apiResponse.status === 200) {
-      dispatchAction(dispatch, VISITS_DATA, apiResponse.data);
+      dispatchAction(dispatch, VISITSDATA, apiResponse.data);
     }
   } catch (error) {
     handleNetworkError(error);
   }
 };
 
 // Add new visit data
-export const ADD_VISIT = (visit, onClose) => async (dispatch) => {
+export const ADD_VISIT = (visit, resetForm) => async (dispatch) => {
   try {
-    const apiResponse = await ApiService.post(`/visits`, visit);
+    const apiResponse = await ApiService.post(`/appointments`, visit);
     console.log("API response - create visit", apiResponse);
     if (apiResponse) {
       dispatch(GET_VISITS());
-      onClose();
+      resetForm();
```

Comment:
I have noticed that reset form is not resetting visitor and date field. please note and fix this as part of future enhancement.

Reply:
Yes Pratyusha, Even I noticed and facing issues with it. I am researching more on how to achieve it.